src/main/resources/static/api.yaml

```diff
@@ -1207,6 +1207,33 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+  # Sessions
+  /logout:
+    post:
+      description: |
+        Logout active user by invalidating the access and refresh tokens
+      consumes:
+        - application/x-www-form-urlencoded
```

Comment:
what makes you think that we should use url-encoded?

Reply:
I don't foresee any new param to be added. Hence found form body to be convenient for parsing without adding a model object.

Reply:
I actually meant `form data` not `form-urlencoded` . Will update if you're okay with it.

Reply:
Updated to json